benchmark/matrix_generator/matrix_generator.cpp

```diff
@@ -42,6 +42,11 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include "benchmark/utils/general.hpp"
 
 
+#ifdef GINKGO_BENCHMARK_ENABLE_TUNING
+#include "benchmark/utils/tuning_variables.hpp"
+#endif  // GINKGO_BENCHMARK_ENABLE_TUNING
```

Comment:
Is it needed? we do not use the matrix here

Reply:
Yes it's needed, otherwise the executable will not compile. The library now has the symbols defined as extern but no actual definition of the symbols. Therefore, the executables need to also compile the variables and know the definition of the variables as well.

Reply:
I see